cmd/server/cmd/server.go

```diff
@@ -49,6 +49,7 @@ import (
 	"istio.io/mixer/pkg/expr"
 	"istio.io/mixer/pkg/il/evaluator"
 	"istio.io/mixer/pkg/pool"
+	mixerRuntime "istio.io/mixer/pkg/runtime"
```

Comment:
Is this needed? It doesn't appear that any other package is called `runtime` in this file.

Reply:
When you are reading code, `runtime` can be confusing, it feel like the builtin runtime.